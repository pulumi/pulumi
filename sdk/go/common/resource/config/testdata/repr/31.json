{"my:outer":{"inner":{"nested":{"a":"b","secure":"val"}}}}