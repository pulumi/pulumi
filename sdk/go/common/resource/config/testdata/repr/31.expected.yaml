decrypt:
  my:outer: '{"inner":{"nested":{"a":"b","secure":"val"}}}'
paths:
  my:outer:
    value:
      inner:
        nested:
          a: b
          secure: val
    string: '{"inner":{"nested":{"a":"b","secure":"val"}}}'
    redacted: '{"inner":{"nested":{"a":"b","secure":"val"}}}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABv/4JsAAEFaW5uZXIXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gkkAAQZuZXN0ZWQXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/giUAAgFhBnN0cmluZwwDAAFiBnNlY3VyZQZzdHJpbmcMBQADdmFs
    secure: false
    isObject: true
  my:outer.inner:
    value:
      nested:
        a: b
        secure: val
    string: '{"nested":{"a":"b","secure":"val"}}'
    redacted: '{"nested":{"a":"b","secure":"val"}}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABM/4JJAAEGbmVzdGVkF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4IlAAIBYQZzdHJpbmcMAwABYgZzZWN1cmUGc3RyaW5nDAUAA3ZhbA==
    secure: false
    isObject: true
  my:outer.inner.nested:
    value:
      a: b
      secure: val
    string: '{"a":"b","secure":"val"}'
    redacted: '{"a":"b","secure":"val"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAo/4IlAAIBYQZzdHJpbmcMAwABYgZzZWN1cmUGc3RyaW5nDAUAA3ZhbA==
    secure: false
    isObject: true
  my:outer.inner.nested.a:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:outer.inner.nested.secure:
    value: val
    string: val
    redacted: val
    object: EBAABnN0cmluZwwFAAN2YWw=
    secure: false
    isObject: false
